src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebReportRouteConfig.kt

```diff
@@ -52,7 +52,7 @@ object WebReportRouteConfig : RouteConfig
                     .secure(runReports)
                     .transform(),
             WebEndpoint(
-                    "/report/:name/parameters/",
+                    "/report/:name/config/parameters/",
```

Comment:
I've changed the route for parameters because the previous route actually clashes with `/report/:name/:version` and the wrong route was being selected in the Selenium tests. I'm not sure why it has been working through the browser but fails on Selenium - I think perhaps the order in which the routes are getting mapped might be non-deterministic - I thought that moving the more specific route to be defined before the more general one would fix the issue, but it did not. So I decided that making an unambiguous route would be the easiest fix. Very open to changes to the route path itself as long it remains unique.